src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -2176,14 +2176,14 @@ impl<'a> TypeChecker<'a> {
                 "invalid arguments for search function, query text must be a constant string, but got {}",
                 query_arg
             ))
-            .set_span(query_scalar.span()));
+                .set_span(query_scalar.span()));
         };
         let Some(query_text) = query_expr.value.as_string() else {
             return Err(ErrorCode::SemanticError(format!(
                 "invalid arguments for search function, query text must be a constant string, but got {}",
                 query_arg
             ))
-            .set_span(query_scalar.span()));
+                .set_span(query_scalar.span()));
```

Comment:
there are many formatting changes in this?
it seems not what u intended from your commit messages.
and ci-check not complaint about them.
is this expected?
will it cause these lines to be changed back and forth in prs?

Reply:
Weird.
Here I respect: `cargo fmt --all` with  rustfmt.toml.